fe/fe-core/src/main/java/org/apache/doris/cloud/load/CloudCopyJob.java

```diff
@@ -1,106 +0,0 @@
-// Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
Maybe use CloudCopyJob is better then CopyJob ?  cause it's only use in cloud mode.